qiskit_nature/__init__.py

```diff
@@ -46,6 +55,7 @@
    properties
    results
    runtime
+   settings
```

Comment:
Why do you add this twice?

We need to be a bit careful here because we actually do `from .settings import settings` because the latter is a "singleton" instance of `QiskitNatureSettings`

Reply:
The `settings` weren't documented anywhere. I first added to modules the thought having it in a Globals section would be better and forgot to remove it from the modules.